ravenpy/models/emulators.py

```diff
@@ -114,13 +114,17 @@ def derived_parameters(self):
             soil0 = self.rvd.GR4J_X1_hlf if self.rvc.soil0 is None else self.rvc.soil0
             soil1 = self.rvc.soil1
 
+        # subbassin_id -> has at least one LakeHRU
+        sb_is_lake = defaultdict(lambda: False)
```

Comment:
Better name might be `sb_contains_lake`.

Reply:
You greatly activated my special fiber for "variables should have semantically accurate names", thanks and done!